front/lib/plans/plan_codes.ts

```diff
@@ -8,6 +8,17 @@ export const TRIAL_PLAN_CODE = "TRIAL_PLAN_CODE";
 
 // Current pro plans:
 export const PRO_PLAN_SEAT_29_CODE = "PRO_PLAN_SEAT_29";
+export const PRO_PLAN_OUTLEAP_CODE = "PRO_PLAN_OUTLEAP";
+export const PRO_PLAN_LARGE_FILES_CODE = "PRO_PLAN_LARGE_FILES";
+
+export const PRO_PLANS_CODES = [
+  PRO_PLAN_SEAT_29_CODE,
+  PRO_PLAN_OUTLEAP_CODE,
+  PRO_PLAN_LARGE_FILES_CODE,
+];
+export const isProPlanCode = (code: string): boolean => {
+  return PRO_PLANS_CODES.includes(code);
```

Comment:
If we're using startsWith('ENT_') for enterprise, can we use startsWith('PRO_') for pro plans for coherence? Or risky somehow?

Reply:
We could but in the Slack thread we say we wanted an array of pro plan codes. 
The reason for which we can't do that for ENT is we expect a lot of them (one per Enterprise customers), while we want to keep the Pro plans under control.